packages/voorbeeld-design-tokens/documentation/theme-builder/theme-builder-step1.stories.tsx

```diff
@@ -0,0 +1,127 @@
+// import { cssVariable } from '@nl-design-system-unstable/theme-toolkit/src/util';
+import { fontFamilystep as voorbeeldfontfamilystep } from './voorbeeld-theme';
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  Button,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Heading3,
+  Heading4,
+  Heading5,
+  Heading6,
+  Link,
+  Paragraph,
+  NavBar,
+} from '@utrecht/component-library-react/dist/css-module';
+import { PropsWithChildren } from 'react';
+
+interface WrapperProps {
+  cssVariables: { [index: string]: any };
+}
+const Wrapper = ({ children, cssVariables }: PropsWithChildren<WrapperProps>) => (
+  <div className="voorbeeld-theme">
```

Comment:
Als je de basic theme als basis wilt gebruiken moet je hier `basic-theme` gebruiken in plaats van voorbeeld theme

Reply:
Yes, `basic-theme` in the making!